### PR TITLE
Add note on Hardhat

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
# Description

Adds a note on Hardhat to network docs

## Issue(s) fixed

* Refers to issue raised in [feedback widget](https://docs.google.com/spreadsheets/d/1UTgpxjEpttRaXbSs_jh8EElecAi89e65bPS2d-lCoOg/edit#gid=1155377714)

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

File: wallet/how-to/manage-networks/add-network.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ sidebar_position: 2
 
 # Add a network
 
+:::note
+See the [Connect to Hardhat Network](../how-to/run-devnet.md) documentation to run a development network using Hardhat and connect MetaMask to it.
+:::
+
```
Comment (alexandratran) [inline wallet/how-to/manage-networks/add-network.md:11]: I would make the note less prominent - maybe move to after the screenshots. And update it to something like the following for simplicity:

```
:::note
You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md).
:::
```

Comment (Montoya) [reply wallet/how-to/manage-networks/add-network.md]: Can the messaging be a bit more general? 

```
Looking for information on how to add a local development network to MetaMask, such as Hardhat? [Learn how to connect a development network.](...)
```

File: wallet/how-to/manage-networks/add-network.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ The confirmations look like the following:
     </div>
 </div>
 
+:::note
+
+You can also [add a Hardhat development network to MetaMask](../how-to/run-devnet.md)
```
Comment (alexandratran) [inline wallet/how-to/manage-networks/add-network.md:35]:
```suggestion
You can also [add a development network](../how-to/run-devnet.md), such as Hardhat, to MetaMask.
```

To address Christian's comment on generalizing the note.

Comment (joaniefromtheblock) [reply wallet/how-to/manage-networks/add-network.md]: Hi @alexandratran I proposed a slightly different wording.

File: wallet/how-to/manage-networks/add-network.md [hunk 3]
```diff
@@ -30,6 +30,12 @@ The confirmations look like the following:
     </div>
 </div>
 
+:::note
+
+To add a local development network like [Hardhat](https://hardhat.org) to MetaMask, see the [connect a development network](../how-to/run-devnet.md) documentation.
```
Comment (alexandratran) [inline wallet/how-to/manage-networks/add-network.md:35]:
```suggestion
To add a local development network such as [Hardhat](https://hardhat.org) to MetaMask, see [Run a development network](../how-to/run-devnet.md).
```

**Review by alexandratran — APPROVED** [review]

**Comment by Montoya** [comment]
Hi, this link is broken. It goes to: https://docs.metamask.io/wallet/how-to/manage-networks/how-to/run-devnet.md/

It should go to: https://docs.metamask.io/wallet/how-to/run-devnet/

**Pull request merged**